packages/rhythmbox.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Rhythmbox < Package
+class Rhythmbox < Autotools
   description 'Rhythmbox is a music playing application for GNOME.'
   homepage 'https://wiki.gnome.org/Apps/Rhythmbox'
   version '3.4.4'
```

Comment:
Update to [3.4.7](https://download.gnome.org/sources/rhythmbox/3.4/rhythmbox-3.4.7.tar.xz)?

Reply:
Updated on arm.